Dockerfile

```
@@ -0,0 +1,15 @@
+FROM rocker/verse
+
+WORKDIR /report
+
+RUN apt update -y && apt install -y \
+ libcurl4-openssl-dev
+
```

```suggestion

RUN tlmgr install \
    colortbl \
    environ \
    ifluatex \
    ifxetex \
    makecell \
    multirow \
    threeparttable \
    threeparttablex \
    trimspaces \
    ulem \
    varwidth \
    wrapfig \
    xcolor

```

Install relevant TeX packages in build.